docs/SRS-Volere/SRS.tex

```diff
@@ -171,15 +171,27 @@ \subsection{Longevity Requirements}
 
 \section{Operational and Environmental Requirements}
 \subsection{Expected Physical Environment}
-\lips
+\begin{enumerate} [{OE-PE}1.]
+  \item Application should be functional in City of Hamilton, Water Division sites and offices.
+\end{enumerate}
 \subsection{Wider Environment Requirements}
-\lips
+\begin{enumerate} [{OE-WE}1.]
+  \item Application should be functional on Mobile and Desktop web browser environments.
+  \item Application should be able to run on any device with a modern web browser.
```

Comment:
scope this down to just Chrome and Microsoft Edge for now, and if the city wants other browsers we'll ask them.

Reply:
Scoped it down to Chrome, Edge and Mobile Browsers.